loaders/clickhouse/append_l2g.sql

```diff
@@ -0,0 +1,3 @@
+insert into ot.l2g_by_slg select * from ot.l2g_log;
+
+insert into ot.l2g_by_gsl select * from ot.l2g_log;
```

Comment:
If everything from l2g_log is inserted into l2g_by_gsl/slg then wouldn't it be better to intermediately insert into l2g_by_gsl/slg without creating l2g_log?

Reply:
But that would mean that we have to change the logic for loading files in the `load_data.sh` script: `load_data_type "l2g" "l2g"`

This would have to be different when loading from scratch or appending data, I think.

Reply:
hmm I see :+1: